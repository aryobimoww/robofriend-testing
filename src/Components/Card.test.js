import {shallow} from 'enzyme';
import React from 'react';
import Card from './Card';
it('expect to render card component', ()=>{
    const wrapper = (shallow(<Card/>));
    expect(wrapper.debug()).toMatchSnapshot();
})
