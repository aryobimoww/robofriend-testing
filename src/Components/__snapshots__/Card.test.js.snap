// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render card component 1`] = `
"<div className=\\"tc bg-washed-green dib br5 pa3 ma3 grow bw2 shadow-5\\">
  <img alt=\\"robots\\" src=\\"https://robohash.org/undefined?size=200x200\\" />
  <div>
    <h2 />
    <p />
  </div>
</div>"
`;
